MovieApp/MovieApp/App/Features/MovieBrowser/Components/MovieSearchBar+Design.swift

```diff
@@ -0,0 +1,139 @@
+        searchTextField.attributedPlaceholder = NSAttributedString(
+            string: "Search",
+            attributes: [NSAttributedString.Key.foregroundColor: UIColor.searchPlaceholderColor,
+                         NSAttributedString.Key.font: UIFont(name: Fonts.proximaMedium, size: 16) ?? .systemFont(ofSize: 16)])
+        searchTextField.font = UIFont(name: Fonts.proximaMedium, size: 16) ?? .systemFont(ofSize: 16)
+        
+        deleteTextButton.isHidden = true
+        deleteTextButton.setImage(ImageEnum.searchDeleteImage.image, for: .normal)
+        deleteTextButton.imageView?.contentMode = .scaleAspectFit
+        deleteTextButton.setTitleColor(.black, for: .normal)
+        
+        cancelButton.isHidden = true
+        cancelButton.setTitle("Cancel", for: .normal)
+        cancelButton.setTitleColor(.primaryBlue, for: .normal)
+        cancelButton.titleLabel?.font = UIFont(name: Fonts.proximaMedium, size: 16) ?? .systemFont(ofSize: 16)
+    }
+    
+    func defineLayoutForViews() {
+        snp.makeConstraints {
+            $0.height.equalTo(searchBarHeight)
+        }
+        
+        searchGrayFieldView.snp.makeConstraints {
+            $0.leading.equalToSuperview().offset(4 * offset)
+            $0.trailing.equalToSuperview().inset(4 * offset)
+            $0.top.bottom.equalToSuperview()
+        }
+        
+        searchBarIconImageView.snp.makeConstraints {
+            $0.leading.equalToSuperview().offset(3 * offset)
+            $0.centerY.equalToSuperview()
+            $0.width.equalTo(0.5 * searchBarHeight)
+        }
+        
+        searchTextField.snp.makeConstraints {
+            $0.leading.equalTo(searchBarIconImageView.snp.trailing).offset(3 * offset)
+            $0.trailing.equalToSuperview().inset(3 * offset)
+            $0.centerY.equalToSuperview()
+            $0.height.equalTo(0.5 * searchBarHeight)
+        }
+        
+        cancelButton.snp.makeConstraints {
+            $0.trailing.equalToSuperview().inset(4 * offset)
+            $0.centerY.equalToSuperview()
+        }
+        
+        deleteTextButton.snp.makeConstraints {
+            $0.trailing.equalToSuperview().inset(2 * offset)
+            $0.centerY.equalToSuperview()
+            $0.height.equalTo(deleteTextButtonHeight)
+        }
+    }
+    
+    func changeCancelButtonVisibility() {
+        if(cancelButton.isHidden) {
+            UIView.animate(withDuration: 0.5, animations: {
+                self.searchGrayFieldView.snp.remakeConstraints {
+                    $0.leading.equalToSuperview().offset(4 * self.offset)
+                    $0.trailing.equalTo(self.cancelButton.snp.leading).offset(-4 * self.offset)
+                    $0.top.bottom.equalToSuperview()
+                }
+                self.deleteTextButton.isHidden.toggle()
+                self.searchGrayFieldView.superview?.layoutIfNeeded()
+            }, completion: { _ in
+                self.cancelButton.isHidden.toggle()
+            })
```

Comment:
This should be:
```swift
UIView.animate(
    withDuration: 0.5,
    animations: {
        // animations block
    },
    completions: {
        // completion block
    })
```

Reply:
I [formated](https://github.com/lihter/Movie-App/pull/24/files#diff-6a7e0c3cc088b3a980b8a92c08ad5d190f909d67f14aa13f150bdc992ab52302R95) the code as advised.